### PR TITLE
[Fix] fix ncnn torch 1.12 dev-1.x

### PR DESCRIPTION
**Pull request by hanrui1sensetime**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

The same as #1430 

## Modification

Same as #1430 

## BC-breaking (Optional)

None.

## Use cases (Optional)

None.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: csrc/mmdeploy/backend_ops/ncnn/onnx2ncnn/fuse_pass.cpp [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+                   std::map<std::string, onnx::TensorProto>& weights,
+                   std::map<std::string, int>& node_reference, std::set<std::string>& blob_names,
+                   int& reduced_node_count) {
+  const int node_count = mutable_graph->node_size();
+  for (int i = 0; i < node_count; ++i) {
+    onnx::NodeProto* node = mutable_graph->mutable_node(i);
+    for (int j = 0; j < node->input_size(); ++j) {
+      std::string output_name = node->input(j);
+      onnx::NodeProto* last_node = find_node_by_output_name(mutable_graph, output_name);
+      fprintf(stderr, "i %d j %d line 14!\n", i, j);
```
Comment (tpoisonooo) [inline csrc/mmdeploy/backend_ops/ncnn/onnx2ncnn/fuse_pass.cpp]: remove useless debug

File: csrc/mmdeploy/backend_ops/ncnn/onnx2ncnn/fuse_pass.cpp [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+void fuse_identity(onnx::GraphProto* mutable_graph,
+                   std::map<std::string, onnx::TensorProto>& weights,
+                   std::map<std::string, int>& node_reference, std::set<std::string>& blob_names,
+                   int& reduced_node_count) {
+  const int node_count = mutable_graph->node_size();
+  for (int i = 0; i < node_count; ++i) {
+    onnx::NodeProto* node = mutable_graph->mutable_node(i);
+    for (int j = 0; j < node->input_size(); ++j) {
+      std::string output_name = node->input(j);
+      onnx::NodeProto* last_node = find_node_by_output_name(mutable_graph, output_name);W
```
Comment (tpoisonooo) [inline csrc/mmdeploy/backend_ops/ncnn/onnx2ncnn/fuse_pass.cpp]: remove tail `;W`

File: csrc/mmdeploy/backend_ops/ncnn/onnx2ncnn/fuse_pass.cpp [hunk 3]
```diff
@@ -1,6 +1,30 @@
 // Copyright (c) OpenMMLab. All rights reserved.
 #include "fuse_pass.h"
 
+void fuse_identity(onnx::GraphProto* mutable_graph,
```
Comment (tpoisonooo) [inline csrc/mmdeploy/backend_ops/ncnn/onnx2ncnn/fuse_pass.cpp:4]: Add some text graph comments, for example:
```c++
fuse  

  identity -->  op  
  
 to   

 noop_reducencnn --> op
```

**Review by tpoisonooo — APPROVED** [review]
LGTM.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**